### ISSUE TITLE
Zombies can't break wooden doors in difficulty hard 

### ISSUE THREAD

**Issue by thibaulthenry** [SpongePowered/Sponge#2498]
Hello there! 

- SpongeVanilla version: 1.12.2-7.1.9 Stable Recommended
- Java version: 8
- Operating System: macOS
- Plugins: only my plugin in development (cosmos) 

I'm currently writting a new world management plugin / per-world management. 
In doing so, I'm testing a bunch of behaviors in Minecraft with Sponge Vanilla as server.

After some tests, it seems like per-world difficulty is working fine.
But I encountered an issue while working with the difficulty set to hard in a world.
Indeed, in this hard level, Zombies should be able to break wooden doors but they actually can't in Sponge Vanilla.

Notice that the gamerule mobGriefing was set to true to allow mobs to grief blocks.

I didn't test every specificities of each difficulty level but if I find anything else I'll edit this post.

Hope this can help!

**Comment by JBYoshi** [comment]
I did some testing on the latest SpongeVanilla, and the first few zombies I spawned could not break doors. However, after spawning several more, one of them did break the door. Whether a particular zombie is able to break a door depends on a random value.

Try spawning a bunch of zombies near the door, or run `/summon minecraft:zombie ~ ~ ~ {CanBreakDoors:1b}` to spawn a zombie that overrides that random value - do either of those work?
